django/pyrog/models.py

```diff
@@ -115,7 +115,9 @@ class StaticInput(Input):
 
 
 class SQLInput(Input):
-    input_group = models.ForeignKey(InputGroup, related_name="sql_inputs", on_delete=models.CASCADE)
+    input_group = models.ForeignKey(
+        InputGroup, blank=True, null=True, related_name="sql_inputs", on_delete=models.CASCADE
```

Comment:
Should we add a constraint telling that an SQLInput comes from whether an InputGroup, a Filter, or a Condition?

Reply:
we will do that when (if) we move the OneToOneRelationship between filter <> sql_input and condition <> sql_input to the sql_input table.